### ISSUE TITLE
Permission denied error when starting API

### ISSUE THREAD

**Issue by yobroyem0** [uwrit/leaf#350]
I run into an error whenever I start the Leaf API as a service on Linux. 
`systemctl status leaf_api.service`
```● leaf_api.service - Leaf API Service
   Loaded: loaded (/var/opt/leafapi/services/leaf_api.service; linked; vendor preset: disabled)
   Active: failed (Result: exit-code) since Mon 2020-07-27 16:49:27 UTC; 6s ago
  Process: 29828 ExecStart=/usr/bin/dotnet API.dll (code=exited, status=140)
 Main PID: 29828 (code=exited, status=140)

Jul 27 16:49:27 leaf-web-server dotnet[29828]: realpath(): Permission denied
Jul 27 16:49:27 leaf-web-server dotnet[29828]: realpath(): Permission denied
Jul 27 16:49:27 leaf-web-server dotnet[29828]: realpath(): Permission denied
Jul 27 16:49:27 leaf-web-server dotnet[29828]: Error:
Jul 27 16:49:27 leaf-web-server dotnet[29828]: An assembly specified in the application dependencies manifes...und:
Jul 27 16:49:27 leaf-web-server dotnet[29828]: package: 'Composure', version: '1.1.0'
Jul 27 16:49:27 leaf-web-server dotnet[29828]: path: 'lib/netcoreapp2.2/Composure.dll'
Jul 27 16:49:27 leaf-web-server systemd[1]: leaf_api.service: main process exited, code=exited, status=140/n/a
Jul 27 16:49:27 leaf-web-server systemd[1]: Unit leaf_api.service entered failed state.
Jul 27 16:49:27 leaf-web-server systemd[1]: leaf_api.service failed.
Hint: Some lines were ellipsized, use -l to show in full.```

I've attempted increasing the permissions on the service account running the service and I've also attempted running the command as root.

**Comment by ndobb** [comment]
@yobroyem0 thanks for posting your issue. The error seems to be calling out the `Composure` lib dependency specifically. 

What version of Leaf are you using? In 3.7 (fixed in 3.7.1) there was a [known issue for pulling that dependency at compilation time](https://github.com/uwrit/leaf/issues/343). Perhaps give that a try.

**Comment by ndobb** [comment]
@yobroyem0 were you able to solve this? If so I'm going to close this issue.
